include/mbedtls/check_config.h

```diff
@@ -774,6 +774,14 @@
 #error "MBEDTLS_HAVE_INT32/MBEDTLS_HAVE_INT64 and MBEDTLS_HAVE_ASM cannot be defined simultaneously"
 #endif /* (MBEDTLS_HAVE_INT32 || MBEDTLS_HAVE_INT64) && MBEDTLS_HAVE_ASM */
 
+#if defined(MBEDTLS_PKCS11_C)
```

Comment:
This should be moved together with the other test involving `MBEDTLS_PKCS11_C`.

Reply:
Done.